README.md

```diff
@@ -168,6 +178,14 @@ Do it as soon as you incorporate your company.
 During the appointment they will scan some documents, ask a few generic questions and try to sell you extra services you most likely don't need. 
 It took 2 days after the appointment to open my account.
 
+### Paying yourself
+
+The most tax efficient (and absolutely legal) way of paying yourself in the UK (assuming no other sources of income) is as follows:
+1. A base salary up to approximately 10000GBP (this amount is adjusted every year by HMRC) which is basically tax free.
+2. Pay the rest in dividends.
+3. Computer equipment, software, travel accommodation, can all be treated as expenses so while technically not income, you not have to use 1 and 2 for the things you need for work.
+Your accountant can and should also advise you on that.
```

Comment:
I think these are very important bits of information, as when you start your contract you will need to know how to pay your self efficiently. This may also aid making the decision to jump ship.